devtools/materialize_asset.py

```diff
@@ -19,33 +27,23 @@ def _parse():
 def main(asset_id):
     """Entry point.
 
-    Defines dagster context like in etl/__init__.py - needs to be kept in sync.
+    Materialize one asset & its upstream deps in-process so you can debug.
 
-    Then creates a job with asset selection.
+    Args:
+        asset_id: Name of asset you want to materialize.
     """
-    etl_fast_settings = EtlSettings.from_yaml(
-        importlib.resources.files("pudl.package_data.settings") / "etl_fast.yml"
-    ).datasets
-
-    # TODO (daz/zach): maybe there's a way to do this directly with dagster cli?
-    defs = Definitions(
-        assets=etl.default_assets,
-        resources=etl.default_resources,
-        jobs=[
-            define_asset_job(
-                name="materialize_one",
-                selection=AssetSelection.keys(asset_id).upstream(),
-                config={
-                    "resources": {
-                        "dataset_settings": {
-                            "config": etl_fast_settings.dict(),
-                        },
-                    },
-                },
-            ),
-        ],
+    materialize(
```

Comment:
@jdangerx @zschira is there a reason why y'all didn't use `materialize()` when you initially created this script?

Reply:
Nope! This is better.

Reply:
Though I think this whole script could probably be replaced with `dagster asset materialize`, like:

```bash
$ dagster asset materialize -m pudl.etl --select "raw_eia860__fgd_equipment"
```

Which works on this branch but fails with the `This AssetsDefinition does not support subsetting.` error on `dev`.

Reply:
True but I'm not sure you can feed the CLI command run config like you can with the `materialize()` function which is helpful to running subsets of years:

```
Usage: dagster asset materialize [OPTIONS]

  Execute a run to materialize a selection of assets

Options:
  -a, --attribute TEXT          Attribute that is either a 1) repository or job or 2) a function that returns a
                                repository or job
  --package-name TEXT           Specify Python package where repository or job function lives
  -m, --module-name TEXT        Specify module where dagster definitions reside as top-level symbols/variables and
                                load the module as a code location in the current python environment.
  -f, --python-file PATH        Specify python file where dagster definitions reside as top-level symbols/variables
                                and load the file as a code location in the current python environment.
  -d, --working-directory TEXT  Specify working directory to use when loading the repository or job
  --select TEXT                 Asset selection to target  [required]
  --partition TEXT              Asset partition to target
  -h, --help                    Show this message and exit.
```

Also, can we use the vs code debugger with the CLI command?